### PR TITLE
Feature notebook mvp

### PR DESCRIPTION
**Pull request by brobuck87**

### CODE REVIEW

File: assets/js/notebook.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  const brewDetails = {
+    grind: grindEl.val(),
+    bean: beanEl.val(),
+    temp: tempEl.val(),
+    ratio: ratioEl.val(),
+  };
+
+  // add object to array from form
+  console.log(pastBrewDetails);
```
Comment (johnsickels) [inline assets/js/notebook.js]: might want to remove this log

File: assets/js/notebook.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  pastBrewDetails.push(brewDetails);
+
+  localStorage.setItem("brewArray", JSON.stringify(pastBrewDetails));
+  buildTable();
+});
+
+function buildTable() {
+  // ADD TRASHCAN IN FUTURE PR FOR USER DELETION
```
Comment (johnsickels) [inline assets/js/notebook.js]: user deletion??? XD

File: index.html [hunk 3]
```diff
@@ -255,10 +255,15 @@
...
     </div>
+<<<<<<< HEAD
+    <div class="grid px-5 grid-flow-cols ">
+      <div class="col-span-3  mt-20">
+=======
 
     <div class="px-10 flex flex-col md:flex-row justify-center p-16">
       <!-- brew -->
       <div class="p-10">
+>>>>>>> main
```
Comment (johnsickels) [inline index.html]: More merge conflict trash

**Comment by brobuck87** [comment]
@johnsickels Should be all cleaned up and ready to merge

**Review by johnsickels — APPROVED** [review]
Looks 💯

**Pull request merged**